/usr/bin/ld: libpoly.a(controller.cpp.o):(.bss+0x8): multiple definition of `polybar::socket_util::make_unix_connection'; libpoly.a(bar.cpp.o):(.bss+0x0): first defined here

### I'm using the AUR package
```yay -S polybar```
### error
```
/usr/bin/ld: libpoly.a(controller.cpp.o):(.bss+0x8): multiple definition of `polybar::socket_util::make_unix_connection'; libpoly.a(bar.cpp.o):(.bss+0x0): first defined here
/usr/bin/ld: libpoly.a(bspwm.cpp.o):(.bss+0x0): multiple definition of `polybar::socket_util::make_unix_connection'; libpoly.a(bar.cpp.o):(.bss+0x0): first defined here
/usr/bin/ld: libpoly.a(bspwm.cpp.o):(.bss+0x0): multiple definition of `polybar::socket_util::make_unix_connection'; libpoly.a(bar.cpp.o):(.bss+0x0): first defined here
/usr/bin/ld: libpoly.a(i3.cpp.o):(.bss+0x0): multiple definition of `polybar::socket_util::make_unix_connection'; libpoly.a(bar.cpp.o):(.bss+0x0): first defined here
/usr/bin/ld: libpoly.a(socket.cpp.o):(.bss+0x0): multiple definition of `polybar::socket_util::make_unix_connection'; libpoly.a(bar.cpp.o):(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [bin/CMakeFiles/polybar.dir/build.make:115: bin/polybar] Error 1
make[1]: *** [CMakeFiles/Makefile2:625: bin/CMakeFiles/polybar.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
Error making: polybar-git
```

### Environment:
* Arch-linux
* WM: i3-gaps

--- a/include/utils/socket.hpp
+++ b/include/utils/socket.hpp
@@ -40,9 +40,9 @@ namespace socket_util {
    *   conn->receive(...);
    * \endcode
    */
-  auto make_unix_connection = [](string&& path) -> unique_ptr<unix_connection> {
+  inline unique_ptr<unix_connection> make_unix_connection(string&& path) {
     return factory_util::unique<unix_connection>(forward<string>(path));
-  };
+  }
 }
 
 POLYBAR_NS_END

